_issues/establishment_failed.md

```diff
@@ -1,29 +1,29 @@
+**A**: The Obama administration fought for legislation to create jobs, close the gender pay gap, help people with "underwater" mortgages keep their houses, and provide access to health coverage for millions of Americans.
 
-* The Temporary Assets Relief Program (TARP) saved the financial and banking industries, and under President Obama's leadership, [virtually all of the "bailout" money was paid back](http://www.politifact.com/new-hampshire/statements/2012/oct/25/barack-obama/barack-obama-says-banks-paid-back-all-federal-bail/).
+* President Obama pushed through the [American Recovery and Reinvestment Act](http://www.nytimes.com/2014/02/23/opinion/sunday/what-the-stimulus-accomplished.html?_r=0). The stimulus package prevented the United States from dipping back into recession. It **created 1.6 million jobs a year for 4 years**, and raised the nation's economic output by 2 to 3 percent from 2009 to 2011.
 
-* The [Home Affordable Refinance Program (HARP)](http://www.harp.gov) and the [Making Home Affordable Program](https://www.makinghomeaffordable.gov/pages/default.aspx) help people impacted by the mortgage crisis keep their homes. 
+* President Obama's first official act was to sign the [Lilly Ledbetter Fair Pay Act](https://www.whitehouse.gov/the-press-office/2016/01/29/fact-sheet-new-steps-advance-equal-pay-seventh-anniversary-lilly) into law, which helped to **close the pay gap between men and women.**
 
-* And perhaps President Obama's crowning achievement, the Affordable Care Act (informally known as "Obamacare"), [provides health insurance to 4.2 million people](http://www.politico.com/story/2014/03/obamacare-affordable-care-act-104540)!
+* The U.S. Treasury created the [Home Affordable Refinance Program (HARP)](http://www.harp.gov) and the [Making Home Affordable Program](https://www.makinghomeaffordable.gov/pages/default.aspx). Both programs **help people impacted by the mortgage crisis keep their homes**. 
 
-So, are people better off now than they were in 2009? [Most Americans are](http://www.nytimes.com/2016/05/01/magazine/president-obama-weighs-his-economic-legacy.html?_r=0). Here are some facts about the health of our economy:
+* And perhaps President Obama's crowning achievement, the Affordable Care Act (informally known as "Obamacare"), [**provides health insurance to 4.2 million people**](http://www.politico.com/story/2014/03/obamacare-affordable-care-act-104540)!
```

Comment:
More than 11.3 million Americans as of January of this year: https://www.washingtonpost.com/news/to-your-health/wp/2016/01/07/more-than-11-3-million-americans-signed-up-for-obamacare-report-says/